pageserver/src/basebackup.rs

```diff
@@ -372,6 +399,9 @@ where
                 // In future we will not generate AUX record for "pg_logical/replorigin_checkpoint" at all,
                 // but now we should handle (skip) it for backward compatibility.
                 continue;
+            } else if path == "pg_stat/pgstat.stat" && !normal_shutdown {
+                // Drop statistic in case of abnormal termination (shtiodown checkopint was not written
```

Comment:
```suggestion
                // Drop statistic in case of abnormal termination, i.e. if we're not starting from the exact LSN
                // of a shutdown checkpoint.
```